__TESTS__/unit/toJson/concatenate.toJson.test.ts

```diff
@@ -26,6 +27,32 @@ describe('concatenate.toJson()', () => {
           publicId: 'cat'
         }
       }
-    ]);
+    ]});
+  });
+
+  it('Concatenate image source + prepend + duration + transition', () => {
+    const transformation = new Transformation()
+      .videoEdit(VideoEdit.concatenate(Concatenate.imageSource('dog').format(Format.jpg()))
+        .prepend()
+        .duration(1)
+        .transition(Concatenate.videoSource('cat'))
+      );
+    expect(transformation.toJson()).toStrictEqual({actions:[
+      {
+        actionType: 'concatenate',
+        source: {
+          qualifierType: 'ImageSource',
+          sourceType: 'image',
+          publicId: 'dog.jpg'
+        },
+        prepend: true,
+        duration: 1,
```

Comment:
`VideoSource/ImageSource` can also have transformations - can we create a more complicated test example?